libs/libpqxx/Makefile

```diff
@@ -0,0 +1,53 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libpqxx
+PKG_VERSION:=7.3.0
+PKG_RELEASE:=1
+PKG_INSTALL:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/jtv/libpqxx
+PKG_SOURCE_VERSION:=$(PKG_VERSION)
+PKG_MIRROR_HASH:=c176771dbbf02e38b84a8f9bf986a83587e2d86d9e0d2238e2a17a036e865951
+
+CMAKE_INSTALL:=1
```

Comment:
One issue I've had with CMAKE_INSTALL in the past is that the pkgconfig paths are totally wrong. See https://github.com/openwrt/packages/blob/master/libs/pugixml/Makefile#L45 for an example.

Please check that libdir and includedir are correct.

Reply:
I checked the **libpqxx.pc** file generated.

> prefix=/usr
> exec_prefix=${prefix}
> libdir=${prefix}/lib
> includedir=${prefix}/include

So libdir and include dir are parameterized by _prefix_, nothing bad that I see...